src/details/ListingsDetails.js

```diff
@@ -34,50 +45,54 @@ const ListingsDetails = () => {
-                                 data-ride="carousel">
-                                <div className="carousel-inner">
-                                    <div className="carousel-item active">
-                                        <img className="d-block w-100" src={horse1} alt={"img"}/>
-                                    </div>
-                                    <div className="carousel-item">
-                                        <img className="d-block w-100" src={horse1} alt={"img"}/>
-                                    </div>
-                                    <div className="carousel-item">
-                                        <img className="d-block w-100" src={horse1} alt={"img"}/>
-                                    </div>
+                            <div className="row">
+                                <div className="col-md-12">
+
+                                    <Carousel autoplay>
+                                        <div>
+                                            <img className={"w-100"} src={horse1} />
+                                        </div>
+                                        <div>
+                                            <img className={"w-100"} src={horse1}/>
+                                        </div>
+                                        <div>
+                                            <img className={"w-100"} src={horse1} />
+                                        </div>
+                                        <div>
+                                            <img className={"w-100"} src={horse1} />
+                                        </div>
+                                    </Carousel>,
+
                                 </div>
-                                <a className="carousel-control-prev" href="#carouselExampleControls" role="button"
-                                   data-slide="prev">
-                                    <span className="carousel-control-prev-icon" aria-hidden="true"></span>
-                                    <span className="sr-only">Previous</span>
-                                </a>
-                                <a className="carousel-control-next" href="#carouselExampleControls" role="button"
-                                   data-slide="next">
-                                    <span className="carousel-control-next-icon" aria-hidden="true"></span>
-                                    <span className="sr-only">Next</span>
-                                </a>
                             </div>
+
                             <div className="row my-5 pl-3 pr-5">
                                 <div className="mr-auto">
                                     <h5>Listing Heading</h5>
-                                    <p style={{margin: 0, padding: 0}}>$ 33,000</p>
+                                    <p className={"price"}>$ 33,000</p>
```

Comment:
I've done this slightly differently. Instead of passes horse data as a prop I have initialize horseData using useState in the file and gave it an initial state and I have wrote down the code for you to use RestApi inside an useEffect hook (it is commented out), un comment it and delete the initial state I've set in the horseData.